src/preprocess.rkt

```diff
@@ -9,16 +9,24 @@
 
 ;; See https://pavpanchekha.com/blog/symmetric-expressions.html
 (define (find-preprocessing initial specification context)
+  ;; f(x) = f(-x)
   (define even-identities
     (reap [sow]
       (for ([variable (in-list (context-vars context))]
             [representation (in-list (context-var-reprs context))])
-        (with-handlers ([exn:fail:user:herbie? (const (void))])
-          ; TODO: Handle case where neg isn't supported for this representation
+        (with-handlers ([exn:fail:user:herbie? (const #f)])
```

Comment:
It doesn't actually matter, but I recently learned that `void` can take (and ignore) any number of arguments, making it perfect for using as an "ignore-exceptions" handler.